workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PublisherDashboard/pages/Apps/AppCreationFlow/ProvideVersionDetailsPage/ProvideVersionDetailsPage.tsx

```diff
@@ -93,12 +96,24 @@ export function ProvideVersionDetailsPage({
 			({key}) => key === 'dxp-license-usage-type'
 		);
 
-		const targetOption = isDXP ? dxpOption : trialOption;
+		const cloudOption = options.find(
+			({key}) => key === 'cloud-license-usage-type'
+		);
+
+		const targetOption = isDXP
+			? dxpOption
+			: isCloud
+				? cloudOption
+				: trialOption;
```

Comment:
Please check if  `trialOption` is necessary `const trialOption = options.find(({key}) => key === 'trial');`

Reply:
no problem